lib/wings/valkyrie/query_service.rb

```diff
@@ -82,17 +82,11 @@ def find_by_alternate_identifier(alternate_identifier:)
       # @param [Valkyrie::ResourceClass]
       # @return [Array<Valkyrie::Resource>]
       def find_members(resource:, model: nil)
+        return [] unless resource.respond_to?(:member_ids) && resource.member_ids.present?
+        all_members = find_many_by_ids(ids: resource.member_ids)
+        return all_members unless model
```

Comment:
This was rewritten to...
* return as early as possible to avoid expensive processing
* avoid conversions back and forth between resources and af object
* avoid processing of models unless a model filter was passed in
* use `find_many_by_ids` such that when that method is pure valkyrie, this method will also operate as pure valkyrie

Reply:
Great revision. We definitely didn't optimize the method, as we were just trying to get the queries working.